### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by anion155** [JsonHunt/save-commands#11]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.

```
Atom.Object.defineProperty.get (/usr/share/atom/resources/app/src/atom.js:55:13)
Object.executeCommand (/home/anion/.atom/packages/save-commands/lib/atom-save-commands.coffee:107:13)
```

**Comment by joshnhickman** [comment]
With the atom 0.206.0 release, save-commands is disabled by default due to this issue. Is there a plan to address this soon?

**Comment by JsonHunt** [comment]
Hi Josh,

I am working on it now

**Comment by joshnhickman** [comment]
It looks like this was fixed with the 0.6.2 release. Thanks!
